flink-table/flink-table-common/src/main/java/org/apache/flink/table/api/dataview/MapView.java

```diff
@@ -88,11 +85,7 @@
  * }
  *
  * }</pre>
- *
- * @param <K> key type
- * @param <V> value type
```

Comment:
The javadoc still seems appropriate here, though its not really adding any insight